target_chains/cosmwasm/tools/src/ci/deployer/osmosis.ts

```diff
@@ -16,16 +15,10 @@ export type OsmosisHost = {
 };
 
 export class OsmosisDeployer implements Deployer {
-  private chain: Chain;
   constructor(
     private endpoint: string,
     private mnemonic: string,
-  ) {
-    const c = chains.find(({ chain_name }) => chain_name === "osmosis");
-    if (c === undefined)
-      throw new Error("Could not find Osmosis in chain registry");
-    this.chain = c;
```

Comment:
This whole block was warning because `chain` is a private variable that's never consumed, meaning none of this should be needed.  Please someone who knows this package better -- please help take a look and make sure there's nothing hacky that's breaking the static analysis here and actually uses this in some unexpected way...